tests/data/example_config.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "test_run",
+    "user_id": "test_user1@mail.com",
+    "job_id": "BaxDa",
+    "baseurl": "http://localhost:3000",
+    "baseurl_v2": "http://localhost:3000",
+    "job_dir": "./tests/jobs/BaxDa",
```

Comment:
same here `job_dir` shouldn't be data release to the user, it should be something that we do internally. the API can build this directory using the job_id plus a server configuration that specifies the jobs dir.